### PR TITLE
Add SDG cards management section

### PR DESCRIPTION
**Pull request by javierm**
## References

* Depends on pull request #4310

## Objectives

* Make it possible to add cards which will be assigned to SDG phases

## Visual changes

In the SDG content management there's a new homepage section listing and allowing to edit cards for each phase

## Notes

In a future pull request, we'll add the cards to the public SDG homepage

### CODE REVIEW

File: db/sdg.rb [hunk 1]
```diff
@@ -23,3 +23,5 @@
 ].each do |code|
   SDG::Target.where(code: code, goal: SDG::Goal.find_by!(code: code.split(".").first)).first_or_create!
 end
+
+3.times { |kind| SDG::Phase.where(kind: kind).first_or_create! }
```
Comment (Senen) [inline db/sdg.rb:27]: What about looping over `SDG::Phase.kinds.values`? By doing this way, if someone customizes the kinds array definition for any reason won't have to update this line too.

```
SDG::Phase.kinds.values.each { |kind| SDG::Phase.where(kind: kind).first_or_create! }
```

**Review by Senen — COMMENTED** [review]
Hi @javierm, i have to do a manual testing yet but here are a few comments.

What do you think about creating card/s for each phase?

File: app/models/sdg/phase.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class SDG::Phase < ApplicationRecord
+  enum kind: %w[sensitization planning monitoring]
```
[the remark this answers is not in the document]
Comment (javierm) [reply app/models/sdg/phase.rb:2]: There's no way to access this array (it isn't stored in a variable/constant), so it's not needed. And the `enum` method converts this array to a hash and freezes it automatically :smile:.

File: db/sdg.rb (hunk 1 above)
Comment (javierm) [reply db/sdg.rb]: Done!

Comment (javierm) [reply db/sdg.rb]: Updated the `dev_seeds` as well :+1:.

File: app/models/sdg/phase.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (Senen) [reply app/models/sdg/phase.rb:2]: Didn't know that. Thanks for the explanation.

File: app/models/abilities/everyone.rb [hunk 3]
```diff
@@ -29,6 +29,7 @@ def initialize(user)
       can [:search, :comments, :read, :create, :new_comment], Legislation::Annotation
 
       can :read, ::SDG::Goal
+      can :read, ::SDG::Phase
```
Comment (Senen) [inline app/models/abilities/everyone.rb:32]: I noticed there a missing spec for this change.

Comment (javierm) [reply app/models/abilities/everyone.rb:32]: Added :+1:.

**Review by Senen — APPROVED** [review]

**Pull request merged**